### ISSUE TITLE
Excalidraw objects texts dissapear for no reason, randomly

### ISSUE THREAD

**Issue by deajan** [excalidraw/excalidraw#6686]
Hello,

I'm trying excalidraw right now at https://excalidraw.com, and noticed that randomly, my texts disappear when loading saved objects.
I've attached an example saved file, of which every object is supposed to have text.

This is what it looks like when loading the file:
![image](https://github.com/excalidraw/excalidraw/assets/4681318/6e7bf4b2-5275-4661-aabe-cf87d6448cbb)

This is what it looks like when I double click on every object (the text reappears)
![image](https://github.com/excalidraw/excalidraw/assets/4681318/8b3bf731-b153-46e4-b1ec-af009ca560bb)

This behavior seems quite random, or at least I cannot find any pattern.

Example file used for the above screenshots:
[Sans-titre-2023-06-14-1117.zip](https://github.com/excalidraw/excalidraw/files/11763778/Sans-titre-2023-06-14-1117.zip)

**Comment by ad1992** [comment]
@deajan in the file you shared I can see the text has `height` set to `0` hence they are not showing up until you try to edit them.

![image](https://github.com/excalidraw/excalidraw/assets/11256141/3949f449-cae0-402c-b1f1-73143194ddba)

Is this a very old drawing which you are trying to load ? because `height` shouldn't be set to `0`. I recall there was earlier a bug due to which this was happening but it was fixed so shouldn't be happening for any drawings

**Comment by deajan** [comment]
I've used excalidraw 0.14.2 for that drawing. Did that version have the bug ?

**Comment by ad1992** [comment]
I can't say 100% as I am not sure why `height` will be set to `0` when you exported it. 
Did you by any chance used brave for exporting earlier?

**Comment by deajan** [comment]
No, I've used excalidraw integrated into another product (being Trilium Notes).
Developper said he doesn't "interact" with excalidraw json exporting, hence the reason I tried to import my drawing at excalidraw.com and realized I got the same bug here.
